api/src/controllers/elasticsearch/utils.js

```diff
@@ -4,13 +4,26 @@ const { capture } = require("../../sentry");
 const ES_NO_LIMIT = 10000;
 
 function searchSubQuery([value], fields) {
+  const words = value?.split(" ");
```

Comment:
Pour enlever les espaces avant et après, au cas où :

```suggestion
  const words = value?.trim().split(" ");
```